geolocator_platform_interface/lib/src/enums/activity_type.dart

```diff
@@ -0,0 +1,21 @@
+/// Represents the possible iOS activity types
```

Comment:
nit: missing a period at the end of the sentence.
```suggestion
/// Represents the possible iOS activity types.
```